### PR TITLE
Open Liberty 20.0.0.3 as a Compatible Software Product for...#855

### LINKED ISSUE #855: Open Liberty 20.0.0.3 as a Compatible Software Product for Jakarta EE 8
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
[Brian Decker - IBM - Eclipse Profile](https://accounts.eclipse.org/users/bdeckerhdt)

**A product name and version**
[Open Liberty 20.0.0.3](https://openliberty.io/)

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**
Both

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
[https://openliberty.io/certifications/assets/OL_logo_navy.zip](https://openliberty.io/certifications/assets/OL_logo_navy.zip)

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
[IBM Corporation](https://www.ibm.com/)

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
Full Platform: [https://github.com/eclipse-ee4j/jakartaee-platform/issues/227](https://github.com/eclipse-ee4j/jakartaee-platform/issues/227)
Web Profile: [https://github.com/eclipse-ee4j/jakartaee-platform/issues/228](https://github.com/eclipse-ee4j/jakartaee-platform/issues/228)

**The URL where Compatible Software Product is available for download**
[https://openliberty.io/downloads/](https://openliberty.io/downloads/)

### PR DESCRIPTION
**Pull request by autumnfound**
Fixes #855

Signed-off-by: Martin Lowe <martin.lowe@eclipse-foundation.org>

### CODE REVIEW

File: data/compatible_products.yml [hunk 1]
```diff
@@ -27,10 +27,10 @@ sets:
 
     - name: Open Liberty
       vendor: IBM Corporation
-      version: 19.0.0.6
+      version: 20.0.0.3
```
Comment (kwsutter) [inline data/compatible_products.yml:30]: I thought we were going to list multiple compatible versions...  This was discussed at a previous Spec Committee call...

File: data/compatible_products.yml [hunk 2]
```diff
@@ -27,10 +27,10 @@ sets:
...
       image: '/images/compatible_products/ibm_open_liberty.png'
       download: 'https://openliberty.io/downloads/'
-      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/439'
+      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/855'
```
Comment (kwsutter) [inline data/compatible_products.yml:33]: I thought these links were supposed to be for the Certification Requests against the Platform repo.  Looking at our Compatibility Page (https://jakarta.ee/compatibility/), we're not consistent.  Some products list the Certification Request (ie. Payara), while others list the Compatible Software Request Issue (ie. Open Liberty).  I think it should be the Certification Request, which in this case would be https://github.com/eclipse-ee4j/jakartaee-platform/issues/227 for the Platform.

File: data/compatible_products.yml [hunk 3]
```diff
@@ -78,10 +78,10 @@ sets:
...
```
Comment (kwsutter) [inline data/compatible_products.yml:84]: Similar comment as above.  But, I would expect the link to be for Web Profile Certification Request:  https://github.com/eclipse-ee4j/jakartaee-platform/issues/228

File: data/compatible_products.yml [hunk 4]
```diff
@@ -27,10 +27,10 @@ sets:
 
     - name: Open Liberty
```
Comment (kwsutter) [inline data/compatible_products.yml:28]: Going to make a general comment for this whole page...  We need the data to be consistent for all of the compatible products -- versions, links, etc.

File: data/compatible_products.yml (hunk 1 above)
Comment (shabnammayel) [reply data/compatible_products.yml:30]: @kwsutter I was informed that it was decided that we would show the most recent product release level on the compatible products page and leave it up to the vendor to list all versions.

**Comment by chrisguindon** [comment]
@shabnammayel  Would you recommend that we hold off merging this PR until the community discussed the concerns that @kwsutter is reporting?

**Comment by TanjaObradovic** [comment]
Yes, please go ahead and publish this.

**Comment by kwsutter** [comment]
I'm surprised that this PR was merged as-is without any consideration of my comments...  I know that we're still discussing this on the Spec Committee.  But, just in general, we're not consistent on this compatibility page.  Some of the products information don't match...  @TanjaObradovic?

**Comment by TanjaObradovic** [comment]
@kwsutter I apologize, as I was fast to publish the new version of  Open Liberty 20.0.0.3 as Compatible Product and not being focused on the fact that Proof of compatibility is showing the wrong link! This link should be https://github.com/eclipse-ee4j/jakartaee-platform/issues/228.

**Comment by chrisguindon** [comment]
@kwsutter @TanjaObradovic Is this something we are trying to address via https://github.com/jakartaee/jakarta.ee/pull/870?

**Comment by TanjaObradovic** [comment]
@autumnfound can we please revert this PR? We did not address all concerns in the PR, so best not to merge yet. Apologies for being too fast to approve.

**Comment by chrisguindon** [comment]
Done!
https://github.com/jakartaee/jakarta.ee/pull/871

**Comment by kwsutter** [comment]
Sorry, now I need some education...  These changes were merged and then reverted via PR #871.  But, that leaves this PR in a merged state...  So, how should we track the necessary changes for Issue #855?  Or, should we just use that Issue to document the eventual necessary changes once all of this is figured out?

**Comment by chrisguindon** [comment]
Please let me know if there is a better to revert commits with Github. The only way I know is to create a new PR.

I did re-open https://github.com/jakartaee/jakarta.ee/issues/855 to make sure we don't forget about this issue but we will be waiting for further instruction before working on this again.

**Comment by kwsutter** [comment]
> Please let me know if there is a better to revert commits with Github. The only way I know is to create a new PR.
> 
You're better at git than I am... :-)

> I did re-open #855 to make sure we don't forget about this issue but we will be waiting for further instruction before working on this again.

This sounds good.  Let's see if we can figure out a proper plan through the Spec Committee and then we'll process this new CR.  We're (IBM) not in any big hurry to get this updated.  We just wanted to complete the process.  Thanks for your help and patience!

**Comment by chrisguindon** [comment]
> > Please let me know if there is a better to revert commits with Github. The only way I know is to create a new PR.

It looks like there is a better way to revert! I will know better next time!
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reverting-a-pull-request

> This sounds good. Let's see if we can figure out a proper plan through the Spec Committee and then we'll process this new CR. We're (IBM) not in any big hurry to get this updated. We just wanted to complete the process. Thanks for your help and patience!

+1 Thanks for letting us know! :)

**Pull request merged**